### ISSUE TITLE
lift_curve fails when using pandas DataFrame

### ISSUE THREAD

**Issue by thomasfrederikhoeck** [andersbogsnes/ml_tooling#123]
#### Code Sample, a copy-pastable example if possible

```python
from ml_tooling import BaseClassModel
from sklearn.datasets import load_iris
from sklearn.linear_model import LogisticRegression
import pandas as pd

# Define a new class
x, y = load_iris(return_X_y=True)
x = pd.DataFrame(x)
y = pd.Series(y)

class IrisModel(BaseClassModel):
    def get_prediction_data(self):
        x, _ = x, y
        return x # Return given observation
        
    def get_training_data(self):
        return x, y
    
log_iris = IrisModel(LogisticRegression())
results = log_iris.score_model()

results.plot.lift_curve()

```
#### Problem description

When running this the following error message is given when using pandas DataFrame as input in the model:
```
---------------------------------------------------------------------------
MetricError                               Traceback (most recent call last)
<ipython-input-7-b1ebf243cdfe> in <module>
     20 results = log_iris.score_model()
     21 
---> 22 results.plot.lift_curve()

~\AppData\Local\Continuum\miniconda3\envs\KS\lib\site-packages\ml_tooling\result.py in lift_curve(self, **kwargs)
    324             title = f'Lift Curve - {self._model_name}'
    325             y_proba = self._model.predict_proba(self._data.test_x)[:, 1]
--> 326             return plot_lift_curve(self._data.test_y, y_proba, title=title, **kwargs)

~\AppData\Local\Continuum\miniconda3\envs\KS\lib\site-packages\ml_tooling\plots.py in plot_lift_curve(y_true, y_proba, title, ax)
    301     percents, gains = _cum_gain_curve(y_true, y_proba)
    302     positives = np.where(y_proba > .5, 1, 0)
--> 303     score = metrics.lift_score(y_true, positives)
    304 
    305     ax.plot(percents, gains / percents, label=f'$Lift = {score:.2f}$ ')

~\AppData\Local\Continuum\miniconda3\envs\KS\lib\site-packages\ml_tooling\metrics.py in lift_score(y_target, y_predicted)
     31     """
     32     if not isinstance(y_target, np.ndarray) or not isinstance(y_predicted, np.ndarray):
---> 33         raise MetricError("Input must be a numpy NDArray")
     34 
     35     n = len(y_target)

MetricError: Input must be a numpy NDArray
```
Checklist
---------
- [x] I have updated ml-tooling to the newest version `pip install ml-tooling --upgrade`
- [x] I have checked that this issue does not already exist

**Comment by andersbogsnes** [comment]
Agree - the tests should all be inputting dataframes

**Comment by andersbogsnes** [comment]
closed by #135
